build_debian.sh

```diff
@@ -188,7 +188,20 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y in
     usbutils                \
     pciutils                \
     iptables-persistent     \
-    logrotate
+    logrotate               \
+    grub2-common
+
+# Python tabulate required for sonic-utilities
+python_tabulate_url=http://ftp.ca.debian.org/debian/pool/main/p/python-tabulate/python-tabulate_0.7.7-1_all.deb
+python_template_deb_temp=`mktemp`
+trap_push "rm -f $python_template_deb_temp"
+wget $python_tabulate_url -qO $python_template_deb_temp && {                                  \
+    sudo dpkg --root=$FILESYSTEM_ROOT -i $python_template_deb_temp ||                         \
+    sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install -f; \
+}
+# Sonic utilities
+sudo dpkg --root=$FILESYSTEM_ROOT -i target/debs/sonic-utilities_*.deb || \
+    sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install -f
```

Comment:
This is generic package that has nothing vendor related.
sonic-generic.bin can have it with no problems.